app/components/account-card.js

```diff
@@ -0,0 +1,81 @@
+import Ember from 'ember';
+
+var computed = Ember.computed;
+
+export default Ember.Component.extend({
+	accountManager: Ember.inject.service('accountManager'),
+
+	purecloud: Ember.inject.service('purecloud'),
+
+	active: computed('accountManager.selectedAccount', function () {
+		return this.get('account').status;
+	}),
+
+	lastAccount: computed('accountManager.finalAccount', function () {
+		this.get('accountManager.finalAccount');
+	}),
+
+	profileImg: null,
+
+	me: null,
+
+	environmentShort(region) {
```

Comment:
Nitpicking here but you could use an object mapping base paths to region shorts to shorten up the code and make it more javascriptesque